source/parser.hera

```diff
@@ -80,6 +83,7 @@ ArgumentsWithTrailingCallExpressions
   # Since this is recursive Arguments must consume input to avoid infinite recursion
   # NOTE: Do not allow trailing template literals to match
   Arguments ( Samedent !Backtick CallExpressionRest )*
+  ExplicitArguments ( !Backtick CallExpressionRest )*
```

Comment:
You can remove this

Reply:
`Samedent` would prevent the first rule from matching, right?

Reply:
Since `Samedent` is inside the `*` it will match with zero items in the array if no `Samedent` was present, then that match would shadow this second production.